src/Config/ValueProvider/SourceDirsProvider.php

```diff
@@ -34,13 +34,11 @@ public function __construct(ConsoleHelper $consoleHelper, QuestionHelper $questi
         $this->questionHelper = $questionHelper;
     }
 
-    public function get(InputInterface $input, OutputInterface $output, array $dirsInCurrentDir): array
+    public function get(InputInterface $input, OutputInterface $output, array $dirsInCurrentDir, array $guessedSourceDirs = []): array
```

Comment:
Seems like this parameter `$guessedSourceDirs` is used only in unit test. Is it a mistake?

Reply:
That's correct. Right now there's no other way to give alternative guesses to the  SourceDirsProvider. They're tightly coupled with the SourceDirGuesser.

Do you think we should decouple them right this time?

Reply:
Sorry, I still don't get it. The `$guessedSourceDirs` parameter is not used in the source code, only it the Unit Test. 

https://github.com/sanmai/infection/blob/0a320bcd2dbf4172821d1d7e1a8a9e1fe08e062f/src/Command/ConfigureCommand.php#L60

This is not correct. If it is not used, it should be removed.

> Right now there's no other way to give alternative guesses to the SourceDirsProvider

If you want to mock `SourceDirGuesser` in order to provide custom guessed values for unit tests - yes, it should be decoupled and injected. We should not create additional parameters just for unit tests.